src/timeAgoInWords.ts

```diff
@@ -0,0 +1,154 @@
+type Options = {
+	format?: FormatType;
+	showTime?: boolean;
+};
+
+type FormatType = 'short' | 'med' | 'long';
```

Comment:
just to raise this again – this shouldn't really come into confict with the content design `Format` here should it? i think it will be scoped to this module?

```suggestion
type Format = 'short' | 'med' | 'long';
```